core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -177,10 +177,25 @@ void create(String tableName, NewTableConfiguration ntc)
    * @param tableName
    *          Name of a table to create and import into.
    * @param importDir
-   *          Directory that contains the files copied by distcp from exportTable
+   *          A directory containing the files copied by distcp from exportTable
    * @since 1.5.0
+   *
+   */
+  default void importTable(String tableName, String importDir)
+      throws TableExistsException, AccumuloException, AccumuloSecurityException {
+    importTable(tableName, Set.of(importDir));
+  }
+
+  /**
+   * Imports a table exported via exportTable and copied via hadoop distcp.
+   *
+   * @param tableName
+   *          Name of a table to create and import into.
+   * @param importDirs
+   *          A set of directories containing the files copied by distcp from exportTable
```

Comment:
Is there anything that would be worth mentioning about the metadata zip file and multiple dirs here?